setup.py

```diff
@@ -1,26 +1,38 @@
 from setuptools import setup, find_packages
 
-tests_require = [
+requirements = (
+    'boto3',
+    'colorama',
+    'pyYaml',
+)
+
+tests_requirements = (
     'flake8',
     'coverage',
     'moto==1.3.14',
     'pyfakefs',
     'pytest',
     'pytest-cov'
-]
+)
+
+
+def get_long_description(filename='README.md'):
+    with open(filename) as file:
+        readme = file.read()
+    return readme
+
 
 setup(
     name='cabinets',
     version='0.1.0',
-    description="Place to store many things.",
+    description="A consistent approach to file operations.",
```

Comment:
I like this as a tag-line. Maybe we can make it even cuter, like:
```suggestion
    description="A consistent approach to file operations, anywhere.",
```
or something of the sort that alludes to the fact that it can do file operations on multiple different storage platforms.